assets/src/edit-story/app/history/context.js

```diff
@@ -15,8 +15,8 @@
  */
 
 /**
- * External dependencies
+ * Internal dependencies
```

Comment:
Up to you, but do we want to go with every context use? Or should we first merge 2-3 key ones and then expand?

Reply:
I was thinking that we could do them all in one go, since it's a low-risk change and all the same type of change (so it's easy to code review).

If we do find problems, useContext/useContextSelectorShallow can be trivially modified call the React version to verify if this is the cause.